src/main/flight/pid.c

```diff
@@ -712,6 +720,11 @@ void pidInit(const pidProfile_t *pidProfile)
 #ifdef USE_RPM_FILTER
     rpmFilterInit(rpmFilterConfig());
 #endif
+    pidAdjustmentFactor = 1;
+    pidAdjustmentChannel = pidConfig()->pid_adjust_aux_channel;
+    if (pidAdjustmentChannel) {
+        pidAdjustmentChannel += (NON_AUX_CHANNEL_COUNT-1) ;
```

Comment:
Please put spaces around operators, and avoid unneeded braces. See https://github.com/betaflight/betaflight/blob/master/docs/development/CodingStyle.md.

Reply:
done